### ISSUE TITLE
Apollo GUI

### ISSUE THREAD

**Issue by Azimer** [Azimer/Apollo64#4]
I am thinking though some UI ideas.  Here is my first mock up.  Thoughts?

http://imgur.com/izjT5XG

**Comment by cxd4** [comment]
I really don't know.  When it comes to GUIs the possibilities are endless, which is why I enjoy limiting my options with something fixed like a TUI.

The Image Art should be fun to code though.  I wrote a wrapper of sorts to expose C-style OpenGL over JavaScript's WebGL calls, but I never implemented texturing and bitmaps to it (glTexImage2D, glTexCoordPointer, ...).

In a GUI, I would probably prefer the ROM directory listing to have the most horizontal space, rather than lay it on the side as a vertical column.  Some file names or good names can be very long, and people can name their files anything.  Fixed-width features like the Apollo commands and options to configure etc. I would probably just cram into its own line of space away from the ROM list.  At least that's my traditional viewpoint...it's an old style but benefits.

**Comment by Azimer** [comment]
I don't believe the mockup really exposed what I wanted to do with the layout very well.  I am going to keep this topic open and once I get OpenGL wired up, I will show you what I mean instead of using a 15 minute 2D mock-up.
